packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -52,13 +52,15 @@ export class WorkspaceFrontendContribution implements CommandContribution, MenuC
             if (root) {
                 const rootUri = new URI(root.uri).parent;
                 const rootStat = await this.fileSystem.getFileStat(rootUri.toString());
```

Comment:
Nice catch! I am not aware of any magical children fetching, and I hope it's not the case. I would say, it is just a leftover. And yes, it is redundant.

Reply:
@epatpol Could you get rid of `rootStat`?

Reply:
It's not redundant I think? Or I'm not understand this correctly, but that rootStat is the parent of the current root, so that when you open the file dialog you open one up level like in vscode.